### PR TITLE
Add jquery-endless-scroll@1.8.0 w/ git auto-update

### PR DESCRIPTION
**Pull request by extend1994**
Pull request for issue: #9719

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/fredwu/jquery-endless-scroll/
 * License and its reference: [MIT/GPL-3.0 Dual](https://github.com/fredwu/jquery-endless-scroll/#license)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 36
   - Count of stars: 820
   - Count of forks: 182

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by extend1994** [comment]
@cdnjs/intern3 please help me review this pull request, thank you.

File: ajax/libs/jquery-endless-scroll/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  "version": "1.8.0",
+  "description": "Endless/infinite scrolling/pagination.",
+  "keywords": [
+    "jquery",
+    "endless",
+    "scroll"
+  ],
+  "licenses": [
+    "MIT",
+    "GPL"
```
Comment (sufuf3) [inline ajax/libs/jquery-endless-scroll/package.json]: You should add the vision of the GPL license. This [lib page](https://github.com/fredwu/jquery-endless-scroll/#license) provides the version of the GPL license.
Ref: https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json

**Comment by extend1994** [comment]
@sufuf3 I modified the licenses info, and the page you mentioned seems just a demo page, so I don't take it as `homepage` in `package.json`.
Please help me review it again, thank you!

**Comment by extend1994** [comment]
@cdnjs/intern3 I had some misunderstanding for it. Now I removed jquery. Please help me review this pull request, thank you.

**Comment by extend1994** [comment]
ping @clairetsai818 @sufuf3

**Comment by extend1994** [comment]
@sufuf3 I revised the commit message, please help me review this pull request, thank you.

File: ajax/libs/jquery-endless-scroll/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  "licenses": [
+    "MIT",
+    "GPL-3.0"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/fredwu/jquery-endless-scroll.git",
+    "fileMap": [
+      {
+        "basePath": "js",
```
Comment (pvnr0082t) [inline ajax/libs/jquery-endless-scroll/package.json]: It's not good to use `js` or `css` as basePath.

**Comment by extend1994** [comment]
@pvnr0082t I updated the file structure and commit message, please help me review it, thank you.

**Comment by extend1994** [comment]
ping @cdnjs/intern2

File: ajax/libs/jquery-endless-scroll/package.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+          "js/jquery.endless-scroll*"
+        ]
+      }
+    ]
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/fredwu/jquery-endless-scroll.git"
+  },
+  "author": "fredwu"
```
Comment (maruilian11) [inline ajax/libs/jquery-endless-scroll/package.json]: It's better to use "Fred Wu", not use the username.

**Comment by maruilian11** [comment]
And please update the title.

**Comment by extend1994** [comment]
@maruilian11 Do you PR title? To `Add jquery-endless-scroll w/ git auto-update`? Thanks.

**Comment by maruilian11** [comment]
yes, you need to mention the old versions, too.

**Comment by extend1994** [comment]
@maruilian11 I thought "add old version" only needs to be mentioned in the commit message? According to many closed PR involving `Add <lib> w/ npm/git auto-update` and `Manually add old versions`.

**Comment by maruilian11** [comment]
@extend1994 If you insist, we can leave the origin title, I just use my adding PR way to review every PR.
Just say your opinion if you don't agree with. Let's update the author field.

**Comment by extend1994** [comment]
@maruilian11 Thanks! I updated the "author" field, please help me review it.

**Comment by extend1994** [comment]
Accidentally close the PR and reopen it.

**Comment by maruilian11** [comment]
@extend1994 next time, please said "different file structure with the current auto-update".

**Comment by maruilian11** [comment]
@cdnjs/intern2 Please help me make the double check, thanks.

**Pull request merged**